jumanji/training/configs/config.yaml

```
@@ -1,8 +1,8 @@
 defaults:
     - _self_
-    - env: snake  # [bin_pack, cleaner, connector, cvrp, game_2048, job_shop, knapsack, maze, minesweeper, rubiks_cube, robot_warehouse, snake, sudoku, tsp]
+    - env: multi_cvrp  # [bin_pack, cleaner, connector, cvrp, multi_cvrp, game_2048, job_shop, knapsack, maze, minesweeper, rubiks_cube, robot_warehouse, snake, sudoku, tsp]
```

Could we revert to initial settings?
```suggestion
    - env: snake  # [bin_pack, cleaner, connector, cvrp, multi_cvrp, game_2048, job_shop, knapsack, maze, minesweeper, rubiks_cube, robot_warehouse, snake, sudoku, tsp]
```